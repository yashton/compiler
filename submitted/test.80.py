return 3.014e10