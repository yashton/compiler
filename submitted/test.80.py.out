(KEYWORD return)
(LIT 3.014)
(ID "e10")
(ENDMARKER)
