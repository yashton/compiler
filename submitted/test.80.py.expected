(KEYWORD return)
(LIT 3.014e10)
(ENDMARKER)
